src/lib/structures/game/BinbGame.ts

```diff
@@ -0,0 +1,172 @@
+				} else {
+					// Not everyone got the answer, check if anyone got it
+					embedDescription = doubleGuessers.length
+						? 'Well done to those who got it!'
+						: 'No one got both song and artist! 🙁 Well done if you got one of them!';
+				}
+			} else {
+				embedDescription = 'Nobody guessed any of it! 🙁 That was a hard one!';
+			}
+
+			const [streakLeaderId, streak] = this.streaks.leader ?? [];
+
+			// If there is a streak leader, it must be one of the guessers.
+			const streakLeader = streak && doubleGuessers.find((id) => id === streakLeaderId);
+			if (streakLeader) {
+				embedFooter += ` • <@${streakLeader}> has a streak of ${streak} 🔥`;
+			}
+		}
+
+		if (this.goal && !this.limit) {
+			embedFooter += ` • Playing to ${this.goal} points`;
+		} else if (this.limit && !this.goal) {
+			embedFooter += ` • Playing to ${this.limit} songs`;
+		} else if (this.limit && this.goal) {
+			embedFooter += ` • Playing to ${this.goal} points or ${this.limit} songs`;
+		}
+
+		const { title, author, uri, color } = nowPlaying!.info;
+
+		const embed = createEmbed(embedDescription, EmbedColor.Secondary)
+			.setURL(uri)
+			.setTitle(`That was "${title}" by ${author}`)
+			.addField('Leaderboard', this.leaderboard.compute())
+			.setFooter({ text: embedFooter });
+
+		if (color) {
+			embed.setColor(color);
+		}
+
+		const thumbnail = resolveThumbnail(nowPlaying!.info);
+		if (thumbnail) {
+			embed.setThumbnail(thumbnail);
+		}
+
+		await Promise.all(this.dmAllPlayers({ embeds: [embed] }));
+	}
+
+	public validGuessChannel(channel: TextChannel | DMChannel | PartialDMChannel) {
+		return isDMChannel(channel);
+	}
+
+	protected calcPointsDivisor(songsListenedTo: number) {
+		/*
+		In Binb mode the playersScore includes 1st 2nd & 3rd bonuses.
+		The following makes the significance of this bonus less with less players
+		so that a single player can't farm points in this mode playing by themselves, abusing the 1st place bonus.
+		With 1 player the points are equivalent to StandardGame (bonus is offset)
+		As the number of players size increases, so does the significance of the bonuses.
+		*/
+		return songsListenedTo * (1 + 1.5 / this.players.size);
+	}
+
+	protected getPlayers(vcMembers: Collection<string, GuildMember>) {
+		// TODO: Prompt players to join the game with a button.
+		const basePlayer: Omit<Player, 'id'> = { lastGameEntryTime: Date.now(), totalPlayTime: 0, songsListenedTo: 0 };
+		return new Collection(vcMembers.map<[Snowflake, Player]>((member) => [member.id, { ...basePlayer, id: member.id }]));
+	}
+
+	private guessedBoth(user: Snowflake) {
+		return this.round.primaryArtistGuessers.includes(user) && this.round.songGuessers.includes(user);
+	}
+
+	private everyoneGuessedBoth() {
+		return this.round.primaryArtistGuessers.length === this.players.size && this.round.songGuessers.length === this.players.size;
+	}
+
+	private dmAllPlayers(options: string | MessagePayload | MessageOptions) {
```

Comment:
I'd rather use a full word verb than "dm", also since we usually capitalize initialisms it would have to be "DM" but since PascalCased functions are the worst thing ever I'd rather just avoid the conflict. Also we're never going to use `MessagePayload`, save yourself an import.
```suggestion
	private messageAllPlayers(options: string | MessageOptions) {
```